app/jobs/mailer_job.rb

```diff
@@ -0,0 +1,9 @@
+class MailerJob < ApplicationJob
+  queue_as :default
+
+  def perform(*args)
+    # Do something later
+    puts 'Hola desde mailer'
+    MailerJob.perform_later
```

Comment:
This job is not sending any email, check your logic, you have to call the mailer (the one that sends the email) in here and what you're doing is to call the MailerJob itself, this may cause like a loop of the same job calling itself but not doing anything.